### PR TITLE
gradle to set project-wide properties

### PR DESCRIPTION
**Pull request by dulmandakh**
Change HelloWorld app template to use project-wide properties. See https://developer.android.com/studio/build/gradle-tips#configure-project-wide-properties.

RN community and third-party native module developers adopted this, so it'll make RN package development and usage easy.

CI is green https://circleci.com/gh/dulmandakh/react-native/507

### CODE REVIEW

**Comment by dulmandakh** [comment]
@janicduplessis just resolved the conflict 😄

File: local-cli/templates/HelloWorld/android/build.gradle [hunk 1]
```diff
@@ -30,3 +30,11 @@ allprojects {
         }
     }
 }
+
+ext {
+    buildToolsVersion = "26.0.2"
```
Comment (gengjiawen) [inline local-cli/templates/HelloWorld/android/build.gradle]: Why not 26.0.3

Comment (dulmandakh) [reply local-cli/templates/HelloWorld/android/build.gradle]: fixed, my bad :(

**Comment by dulmandakh** [comment]
bumping targetSdk to 26, no need to touch minSdk because the app won't work on Android versions below minSdk.

**Pull request closed without merging**